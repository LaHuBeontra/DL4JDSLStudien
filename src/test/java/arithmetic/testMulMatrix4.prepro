function main() {
	import mat4 testMat4_1, mat4 testMat4_2;

	mat4 result = testMat4_1 * testMat4_2;

	export result;
}
