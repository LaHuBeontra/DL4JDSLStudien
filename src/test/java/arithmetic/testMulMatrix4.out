DefaultPreProDataSet{
dataSetEntries=
[DataSetEntry{variableName='result', variableShape=[4, 4, 4], startColumn=0, length=16}],
dataSet=
[[180.00,  200.00,  220.00,  240.00,  404.00,  456.00,  508.00,  560.00,  628.00,  712.00,  796.00,  880.00,  852.00,  968.00,  1,084.00,  1,200.00],  
 [3,444.00,  3,592.00,  3,740.00,  3,888.00,  4,180.00,  4,360.00,  4,540.00,  4,720.00,  4,916.00,  5,128.00,  5,340.00,  5,552.00,  5,652.00,  5,896.00,  6,140.00,  6,384.00],  
 [10,804.00,  11,080.00,  11,356.00,  11,632.00,  12,052.00,  12,360.00,  12,668.00,  12,976.00,  13,300.00,  13,640.00,  13,980.00,  14,320.00,  14,548.00,  14,920.00,  15,292.00,  15,664.00],  
 [22,260.00,  22,664.00,  23,068.00,  23,472.00,  24,020.00,  24,456.00,  24,892.00,  25,328.00,  25,780.00,  26,248.00,  26,716.00,  27,184.00,  27,540.00,  28,040.00,  28,540.00,  29,040.00]]

calculated_variables=
result=[[[180.00,  200.00,  220.00,  240.00],  
  [404.00,  456.00,  508.00,  560.00],  
  [628.00,  712.00,  796.00,  880.00],  
  [852.00,  968.00,  1,084.00,  1,200.00]],  

 [[3,444.00,  3,592.00,  3,740.00,  3,888.00],  
  [4,180.00,  4,360.00,  4,540.00,  4,720.00],  
  [4,916.00,  5,128.00,  5,340.00,  5,552.00],  
  [5,652.00,  5,896.00,  6,140.00,  6,384.00]],  

 [[10,804.00,  11,080.00,  11,356.00,  11,632.00],  
  [12,052.00,  12,360.00,  12,668.00,  12,976.00],  
  [13,300.00,  13,640.00,  13,980.00,  14,320.00],  
  [14,548.00,  14,920.00,  15,292.00,  15,664.00]],  

 [[22,260.00,  22,664.00,  23,068.00,  23,472.00],  
  [24,020.00,  24,456.00,  24,892.00,  25,328.00],  
  [25,780.00,  26,248.00,  26,716.00,  27,184.00],  
  [27,540.00,  28,040.00,  28,540.00,  29,040.00]]]
}